dvaas/BUILD.bazel

```diff
@@ -20,6 +20,56 @@ package(
     licenses = ["notice"],
 )
 
+# Since DVaaS's user-facing API is still in development and subject to change, we restrict
+# visibility to an explicit allow list. Send us a CL or come talk to us if you'd like to be added.
+DVAAS_USER_ALLOW_LIST = [
+    "//platforms/networking/gpins/testing/blackbox/p4:__subpackages__",
+    "//platforms/networking/gpins/testing/blackbox/replay:__subpackages__",
+    "//platforms/networking/gpins/testing/ondatra/thinkit/tests:__pkg__",
```

Comment:
platforms/networking are internal dirs, can you remove them ?

Reply:
Hi @kishanps The Changes have been addressed, can you pl review?